src/script/build-integration-branch

```diff
@@ -42,6 +42,14 @@ for issue in j:
     prtext.append(pr['html_url'] + ' - ' + pr['title'])
 print("--- queried %s prs" % len(prs))
 
+# update local master
+print('--- checkout master')
+r = call('git checkout master', shell=True)
+assert not r
+print('--- pull master from upstream')
+r = call('git pull upstream master', shell=True)
```

Comment:
in my case, the ceph/ceph repo is not named `upstream`. not sure about other developers.

Reply:
I made the assumption based on [1], which is widely used among the core developers.
For your case, I think we could simply add a alias, e.g.:
```
$ git remote -v
ceph    https://github.com/ceph/ceph.git (fetch)
ceph    https://github.com/ceph/ceph.git (push)
origin  https://xiexingguo@github.com/xiexingguo/ceph.git (fetch)
origin  https://xiexingguo@github.com/xiexingguo/ceph.git (push)
upstream        https://github.com/ceph/ceph.git (fetch)
upstream        https://github.com/ceph/ceph.git (push)
```

I wouldn't mind if there is any other solutions as long as they can achieve the same effect...

----
[1] https://github.com/ceph/ceph/blob/master/src/script/ptl-tool.py#L15